data/cncf.yaml

```diff
@@ -1010,6 +1010,26 @@
       url: https://github.com/knative/serving
       check_sets:
         - code
+    - name: test-infra
+      url: https://github.com/knative/test-infra
+      check_sets:
+        - code-lite
```

Comment:
This should be `knative/infra` now

Reply:
created https://github.com/cncf/clomonitor/pull/1264 now